public/expressions/constants.ts

```diff
@@ -11,3 +11,5 @@ export const VIS_LAYER_PLUGIN_TYPE = 'Anomaly Detectors';
 export const TYPE_OF_EXPR_VIS_LAYERS = 'vis_layers';
 
 export const OVERLAY_ANOMALIES = 'overlay_anomalies';
+
+export const PLUGIN_EVENT_TYPE = 'anomalies';
```

Comment:
nit: Since this is exposed in the tooltip, we should have it capitalized: 'Anomalies'. Might be worth adding a comment about that here too.

Again, it will be more clear in the docs but we don't have those just yet.